### ISSUE TITLE
Add a wiki page to explain how to run & debug your bot locally while using Direct Line Speech Client

### ISSUE THREAD

**Issue by dargilco** [Azure-Samples/Cognitive-Services-Direct-Line-Speech-Client#22]
This is a common case: you would like to develop your bot while running it locally on your PC. You would like to use Direct Line (DL) Speech Client and Direct Line channel while doing so, as if it was a real deployment.  A wiki is needed to show how you do this.

**Comment by BharathPalanivelu** [comment]
Agreed. Must wanted samples... most of the blogs are half-cooked with only configurations required to do but then not sure what to do next
